src/atopile/buildutil.py

```diff
@@ -42,16 +60,59 @@ def build(build_ctx: BuildContext, app: Module) -> None:
         add_api_pickers(n, base_prio=10)
     pick_part_recursively(app)
 
-    logger.info("Make graph")
-    G = app.get_graph()
-
     logger.info("Running checks")
     run_checks(app, G)
 
     build_ctx.ensure_paths()
 
     logger.info("Make netlist & pcb")
-    apply_design(build_ctx.paths, app, G, transform=None)
+    build_paths = build_ctx.paths
```

Comment:
I've straightened out the build pipeline here by coping in the content of the `libs/app` functions, flattening them a bit and then deleting steps